distribution/lib/Standard/Test/0.0.0-dev/src/Extensions_Helpers.enso

```diff
@@ -0,0 +1,14 @@
+from Standard.Base import all
```

Comment:
@radeusgd please use the `private` keyword!

Yes, I see the usage in `test/Table_Tests/src/Util.enso`. Copy the code! Or turn this into official API. My preference is to copy as just two copies aren't enough to benefit from the hassle of turning this into the API.